dockerfiles/nginx-tls/nginx.conf.tpl

```diff
@@ -20,8 +19,8 @@ http {
 
   server {
     listen       8443 ssl;
-    ssl_protocols       TLSv1.1 TLSv1.2;
-    ssl_ciphers         ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128:AES256:HIGH:DES-CBC3-SHA:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4:!kEDH;
+    ssl_protocols       TLSv1.2 TLSv1.3;
```

Comment:
is this going to cut-off old browsers not supporting 1.2?

Reply:
No, according to Kibana, none of hub components are using TLSv1.2. Nginx is internal and is not facing public Internet directly.

Reply:
I mean TLS v1.1.

Reply:
ah ok, I wasn't sure whether the user traffic was going through nginx or not